### DISCUSSION TITLE
Need help for Flask-SocketIO and Flutter Client

### DISCUSSION THREAD

**Discussion by amougnom** [miguelgrinberg/Flask-SocketIO#1840] [General]
Hello,

I have a Flask-SocketIO server (version 5) working fine with Python clients but can't connect with Flutter socketio_client.

According to many posts, it version issues. I downgrade Flask-SocketIO to 4.3.1 and I can connect but Server can't send message to client and I know it's still version problem.

Please here is my inquiries :

1) Which version of Flask, Flask-SocketIO, python-socketio en python-engineio are recommended to support flutter client 2.0.0Beta
2)  Which version of Flask, Flask-SocketIO, python-socketio en python-engineio are recommended to support flutter client 1.0.2

Thanks for helping please.

Regards.

**Comment by miguelgrinberg** [comment]
I'm sorry, but I cannot answer questions or offer support for the flutter client. I track compatibility against the official JavaScript library, which is the reference implementation.

**Comment by amougnom** [comment]
Hello,

Thanks for reply. I finally found the versions working perfectly together:

Server :

python-socketio    4.6.0
Flask              2.1.2 
Flask-SocketIO     4.3.1
python-engineio    3.13.2
eventlet           0.30.2

Flutter Client

socket_io_client: 1.0.2

Thanks.
